rest_local.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: all
```

Comment:
It might have been more idiomatic to define a localhost manifest file with overloads for the appropriate variables, rather than copy-pasting another playbook. I accept that this might not be the time to refactor the existing playbooks.